src/lib/ZestyExplorer.js

```diff
@@ -161,6 +160,8 @@ export const ZestyExplorer = ({ content }) => {
     letterSpacing: '1px',
     fontFamily: "'Arial Rounded MT Bold','Helvetica Rounded',Arial,sans-serif"
   }
+
+  console.log('open state',open)
```

Comment:
remove por favor

Reply:
thz, im removing it for now, working on that tomorrow